ext/sockets/sockets.c

```diff
@@ -1952,6 +1953,40 @@ PHP_FUNCTION(socket_set_option)
 		}
 #endif
 
+#ifdef SO_ATTACH_REUSEPORT_CBPF
+		case SO_ATTACH_REUSEPORT_CBPF: {
+			convert_to_long(arg4);
+
+			if (!Z_LVAL_P(arg4)) {
+				ov = 1;
+				optlen = sizeof(ov);
+				opt_ptr = &ov;
+				optname = SO_DETACH_BPF;
+			} else {
+                uint32_t k = (uint32_t)Z_LVAL_P(arg4);
+                static struct sock_filter cbpf[8] = {0};
+                static struct sock_fprog bpfprog;
+
+                switch (k) {
+                case SKF_AD_CPU:
```

Comment:
I was thinking if this is the best name as it's a part of actual CBPF. Although it's the part that actual matters so it makes some sense. Basically I couldn't come up with anything better so guess it's fine :)